OpenLIFUData/OpenLIFUData.py

```diff
@@ -1276,6 +1467,35 @@ def add_volume_to_database(self, subject_id: str, volume_id: str, volume_name: s
 
         self.db.write_volume(subject_id, volume_id, volume_name, volume_filepath, on_conflict = openlifu_lz().db.database.OnConflictOpts.OVERWRITE)
   
+    def add_session_to_database(self, subject_id: str, session_parameters: Dict):
+        """ Add new session to selected subject in the loaded openlifu database
+
+        Args:
+            subject: Tuple containing 'name' and 'id' of the selected subject
+            session_parameters: Dictionary containing the parameters output from the CreateNewSession Dialog
+        """
```

Comment:
add `-> bool` type annotation for return value

also describe meaning of return value in the docstring